alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/clustering/ClusteringMetricsBeanPostProcessor.java

```diff
@@ -60,6 +60,7 @@ private void createClusteringMetricsBean(BeanDefinitionRegistry beanDefinitionRe
             constructorArgumentValues.addGenericArgumentValue(meterRegistry);
             constructorArgumentValues.addGenericArgumentValue(globalProperties);
             clusteringMetricsBean.setConstructorArgumentValues(constructorArgumentValues);
+            clusteringMetricsBean.setLazyInit(true);
```

Comment:
does it get picked up now by the MeterBinder scanner?

Reply:
I bleive so. Works locally + I added the metric to the integration test that checks if the metric is there and that passed also